lib/order.rb

```diff
@@ -0,0 +1,29 @@
+require 'menu'
+
+class Order
+  attr_reader :dishes, :menu
```

Comment:
You could wrap these in methods so that you don't expose the state of the object